package.json

```diff
@@ -63,6 +63,7 @@
     "graphql-extensions": "file:packages/graphql-extensions"
   },
   "devDependencies": {
+    "@azure/functions": "^1.0.1-beta2",
```

Comment:
This addition should be correlated with a `package-lock.json` update within this PR, but it's not currently included.  If you run:

    npm install --save-dev @azure/functions@1.0.1-beta2

It should also patch `package-lock.json` file with that addition!

Reply:
Not sure if it's desirable, but there seems to be another newer version now: [`1.0.2-beta1`](https://www.npmjs.com/package/@azure/functions?activeTab=versions).  (Though I tried it, and it didn't fix the other typing concerns I'm about to note below. 😉 )

Reply:
🤷‍♂️ no clue what happened here. Tried installing again locally and got some crazy weird errors of `npm ERR! Maximum call stack size exceeded`, but I just ran `npm run clean` and after that, re-installed packages successfully.